### DISCUSSION TITLE
Script problem after update

### DISCUSSION THREAD

**Discussion by techweb-digimedia** [breezefront/community#325] [1. Help] [answered]
i just updated my magento installation from 2.4.6 to 2.4.7-p2 and Breeze & breeze evolution to 2.17.16 & 2.2.0, i have found that some scripts no longer work:

`$(document).on("breeze:load", function () {
    $.alert({
        title: $t('Title'),
        content: $t('Content')
    });
});`

this script (in extend.js file in child theme) now returns the error "$.alert is not a function", it worked fine in the old version (Breeze 2.9.1 / evolution 2.1.5 ), didn't find any clues in the changelog

**Comment by vovayatsyuk** [comment, marked as answer]
I forgot to mention this change, sorry. It's caused by `define|require` function refactoring.

You need to explicitly require dependency now (like in Luma). Here is how it should work:

```js
$(document).on('breeze:load', function () {
  require(['Magento_Ui/js/modal/alert'], function (alert) {
    alert({ 
      title: $t('Title'), 
      content: $t('Content') 
    }); 
  });
});
```

p.s. I updated the code example to match your use-case.
